File: app/src/main/java/com/example/myapplication/atest/UserGetValidateCode.java
package com.example.myapplication.atest;

import com.google.gson.annotations.SerializedName;

public class UserGetValidateCode {
    public static class RequestParam {
        @SerializedName("Email")
        String email;
        @SerializedName("Language")
        String language;

        public RequestParam(String email) {
            this.email = email;

            //获取系统语言
            language = LanguageUtil.getSystemLanguageForSendEmail();
        }
    }
}

File: app/src/main/java/com/example/myapplication/atestkotlin/UserGetValidateCode.kt
package com.example.myapplication.atestkotlin

import com.example.myapplication.atest.LanguageUtil
import com.google.gson.annotations.SerializedName

/**
 * Copyright (c) 2022 Raysharp.cn. All rights reserved.
 *
 * UserGetValidateCode
 * @author liujianming
 * @date 2022-01-05
 */
class UserGetValidateCode {
    class RequestParam(@field:SerializedName("Email") var email: String) {
        @SerializedName("Language")
        var language: String

        init {

            //获取系统语言
            language = LanguageUtil.getSystemLanguageForSendEmail()
        }
    }
}